build/NuGetPackages/Microsoft.Build.Framework.nuspec

```diff
@@ -15,6 +15,7 @@
     <tags>MSBuild</tags>
     <dependencies>
       <group targetFramework=".NETStandard1.3">
+        <dependency id="System.Resources.ResourceManager" version="4.0.1" />
```

Comment:
Any idea why this was added?  I don't see this dependency in Framework's project.json